package.json

```diff
@@ -382,6 +382,7 @@
     "react-window": "1.8.10",
     "react-window-infinite-loader": "1.0.9",
     "react-zoom-pan-pinch": "^3.3.0",
+    "reduce-reducers": "^1.0.4",
```

Comment:
This might raise some eyebrows so I'll elaborate here for posterity;

Redux's `combineReducers()` will create a _nested_ data structure isolating each reducer by a given "key". This is not what we want, we want separate reducers to modify the _same_ data structure – this is what `reduce-reducers` allows us to achieve.

See [this SO answer](https://stackoverflow.com/questions/38652789/correct-usage-of-reduce-reducers/44371190#44371190).